__tests__/components/Comments/WriteComment.test.js

```diff
@@ -33,9 +33,10 @@ describe('WriteComment should run correctly', () => {
 
   it('styles should run correctly', () => {
     expect(styles.replyToContainer).toEqual({
-      marginLeft: 60,
+      marginLeft: 53,
       alignItems: 'center',
-      flexDirection: 'row'
+      flexDirection: 'row',
+      maxWidth: normalize(150)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style properties for `replyToContainer` have been updated. The `marginLeft` property has been reduced from 60 to 53 and a new `maxWidth` property has been added with a value of `normalize(150)`. Make sure these changes do not affect the layout adversely.
```diff
-      marginLeft: 60,
-      flexDirection: 'row'
+      marginLeft: 53,
+      flexDirection: 'row',
+      maxWidth: normalize(150)
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->